lib/samples/display_scene_from_mobile_scene_package/display_scene_from_mobile_scene_package.dart

```diff
@@ -49,48 +47,15 @@ class _DisplaySceneFromMobileScenePackageState
             controllerProvider: () => _sceneViewController,
             onSceneViewReady: onSceneViewReady,
           ),
```

Comment:
Now that the download indicator is gone, the `_ready` flag is no longer used and the `Stack` widget isn't necessary. So we can delete those and simplify the widget tree.